update-packages.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+set -e
```

Comment:
```suggestion
set -e -u -o pipefail
```
* `-u` - fail when using undefined variables
* `-o pipefail` - fail if one of the commands in a pipe fails

Reply:
`-o pipefail` is not supported by `/bin/sh`. `-u` is added now.